src/sidebar/cross-origin-rpc.js

```diff
@@ -11,7 +25,19 @@ let preStartQueue = [];
  */
 const registeredMethods = store => {
   return {
-    changeFocusModeUser: store.changeFocusModeUser,
+    /**
+     * @param {FocusUserInfo} userInfo
+     */
+    changeFocusModeUser: userInfo => {
```

Comment:
For callbacks / methods with a single parameter and no additional documentation, I tend to save some vertical space by writing the JSDoc on one line. eg. `/** @param {FocusUserInfo} userInfo */`.